### PR TITLE
GH-90 Drop Spigot Support, move to one jar, fix compatibility with other plugins.

### PR DESCRIPTION
**Pull request by vLuckyyy**
According to **Rollczi**, we have determined that maintaining support for **Spigot** in our case is senseless and will significantly complicate our support efforts. In our opinion, this will not inconvenience anyone, as there is currently no server using **Spigot** (via ChatFormatter bStats). 

This pull request also includes the exclusion of **two .jar** files, which have finally been replaced with one. Furthermore, this change likely resolves all conflicts with other plugins.

**Fixes:** GH-90, GH-82

### CODE REVIEW

**Review by imDMK — APPROVED** [review]

File: .github/workflows/gradle.yml [hunk 1]
```diff
@@ -34,14 +34,9 @@ jobs:
       - name: Build with Gradle
         uses: gradle/gradle-build-action@62cce3c597efd445cd71ee868887b8b1117703a7
         with:
-          arguments: core:shadowJar paper-support:shadowJar
+          arguments: shadowAll
```
Comment (Rollczi) [inline .github/workflows/gradle.yml:37]: nie powinno być
`chatformatter-paper-plugin:shadowAll`
?

Comment (vLuckyyy) [reply .github/workflows/gradle.yml:37]: Taska rejestrujesz globalnie, więc to bez różnicy

**Review by Rollczi — APPROVED** [review]
image

**Comment by vLuckyyy** [comment]
> image

**Review by CitralFlo — APPROVED** [review]
image

Question: Should "Available for Spigot" be removed? If yes pls remove it.

**Comment by vLuckyyy** [comment]
> image
> 
> Question: Should "Available for Spigot" be removed? If yes pls remove it.

I update all git-related files on another PR.

**Review by Hyd3r1 — APPROVED** [review]

**Review by igoyek — APPROVED** [review]

**Review by Kamicjusz — APPROVED** [review]

**Review by eripe14 — APPROVED** [review]

**Review by Osnixer — APPROVED** [review]

**Pull request merged**